website/docs/docs/get-started/dbt-cloud-features.md

```diff
@@ -3,44 +3,109 @@ title: "dbt Cloud features"
+    link="/docs/deploy/cloud-ci-job"
+    icon="pencil-paper"/>  
+
+   <Card
+    title="Security"
+    body="Manage risk with SOC-2 compliance, CI/CD deployment, RBAC, and ELT architecture."
+    link="https://www.getdbt.com/security/"
+    icon="pencil-paper"/>  
+
+   <Card
+    title="dbt Semantic Layer*"
+    body="Use the dbt Semantic Layer to define metrics alongside your dbt models and query them from any integrated analytics tool. Get the same answers everywhere, every time."
+    link="/docs/use-dbt-semantic-layer/dbt-semantic-layer"
+    icon="pencil-paper"/>  
+
+   <Card
+    title="Metadata API*"
+    body="Use the Metadata API to enhance your workflow and run ad-hoc queries, browse schema, or query the dbt Semantic Layer. dbt Cloud serves a GraphQL API, which supports arbitrary queries."
+    link="/docs/dbt-cloud-apis/metadata-api"
+    icon="pencil-paper"/> 
+
+
+<Card
+    title="Model timing dashboard*"
+    body="The Model timing dashboard visualizes and lets you explore your run and surface model bottlenecks. The dashboard shows model info, order, and run time for each job completed. The display only appears for successfully completed jobs, and the top 1% of model times are highlighted. Access the dashboard on the Run Overview page in dbt Cloud."
+    link="/docs/dbt-versions/release-notes/January-2022/model-timing-more"
+    icon="pencil-paper"/> 
+</div> <br />
+
+ ***These features are available on [select plans](https://www.getdbt.com/pricing/).**
+
+## Related questions
+
+<details>
+  <summary>How can I contribute to dbt Cloud?</summary>
+  <div>
+    <div>Anyone can contribute to the dbt project. And whether it's a dbt package, a plugin, dbt-core, or this documentation site, contributing to the open source code that supports the dbt ecosystem is a great way to level yourself up as a developer, and give back to the community. See <a href="https://docs.getdbt.com/community/resources/oss-expectations">Contributing</a> for details on what to expect when contributing to the dbt open source software (OSS). </div>
+  </div>
+</details>
+<details>
+  <summary>What type of support is provided with dbt Cloud?</summary>
+  <div>
+    <div>The global dbt Support team is available to dbt Cloud customers by email or in-product live chat. Developer and Team accounts offer 24x5 support, while Enterprise customers have priority access and options for custom coverage. <br></br><br></br>Additionally, Enterprise plan customers receive implementation assistance, dedicated account management, and a dbt Labs Security and Legal review. <br></br><br></br> If you have project-related or modeling questions, review <a href="https://docs.getdbt.com/docs/dbt-cloud/cloud-dbt-cloud-support">our Support page</a> or <a href="http://getdbt.slack.com/">dbt Community Slack</a> to get help as well. </div>
+  </div>
```

Comment:
sugg: removing this question / answer and adding a link to https://docs.getdbt.com/docs/dbt-support#dbt-cloud-support in the **Related docs** section below

Reply:
love the suggestions re: the <details> content, i wasn't keen on them as i felt they were out of place and these suggestions are great!